src/index.ts

```diff
@@ -0,0 +1,15 @@
+export class Challenge {
+    buzz (numero : number) {
```

Comment:
Please always use the **English language** in variable declarations.
```
buzz(entry: number)
{
   // code
}
```

Also always specify access modifiers and return types even when  methods are public as this keep the code more readable.
```
public buzz(entry: number): Promise<string>
{
   // code
}
```